app/client/src/components/pages/Community/components/tabs/IdentifiedIssues.js

```diff
@@ -156,29 +154,25 @@ function IdentifiedIssues() {
     [permittedDischargers, permittedDischargersData],
   );
 
-  const convertFacilityToGraphic = React.useCallback(
-    (facility: Object) => {
-      return new Graphic({
-        geometry: {
-          type: 'point', // autocasts as new Point()
-          longitude: facility['FacLong'],
-          latitude: facility['FacLat'],
-        },
-        attributes: facility,
-      });
-    },
-    [Graphic],
-  );
+  const convertFacilityToGraphic = (facility: Object) => {
+    return new Graphic({
+      geometry: {
+        type: 'point', // autocasts as new Point()
+        longitude: facility['FacLong'],
+        latitude: facility['FacLat'],
+      },
+      attributes: facility,
+    });
+  };
```

Comment:
Since this no longer needs to use React's `useCallback` hook, I'm going to convert it to a function declaration (it's a matter of preference, but I generally prefer function declarations over function expressions)... I actually see it's only called once, so I'm going to move the function's returned value (e.g. the creation of a new graphic) down into the one place it's invoked below.

I'll keep my eye out for other situations where this occurs in other files too, and will refactor those too, as needed.